b2sdk/scan/policies.py

```diff
@@ -140,6 +141,7 @@ def __init__(
         :param exclude_modified_after: optionally exclude file versions (both local and b2) modified after (in millis)
         :param exclude_uploaded_before: optionally exclude b2 file versions uploaded before (in millis)
         :param exclude_uploaded_after: optionally exclude b2 file versions uploaded after (in millis)
+        :param symlink_max_visits: maximum number of times to follow a symlink, to avoid infinite loops (default: 10)
```

Comment:
I don't think users of this class want us to limit this. I think they just want a warning and automatic recovery (skip) whenever an infinite symlink loop is encountered.